src/blob/file.rs

```diff
@@ -50,12 +50,13 @@ impl File {
         self.size.load(ORD)
     }
 
-    pub(crate) async fn write_append(&self, buf: &[u8]) -> IOResult<()> {
+    pub(crate) async fn write_append(&self, buf: &[u8]) -> IOResult<usize> {
```

Comment:
Perhaps `write_at` should be updated in the same way. What do you think?

Reply:
Currently there is no reason, I think it should be updated when needed